prowler/providers/aws/services/rds/rds_instance_multi_az/rds_instance_multi_az.py

```diff
@@ -10,16 +10,27 @@ def execute(self):
             report.region = db_instance.region
             report.resource_id = db_instance.id
             report.resource_tags = db_instance.tags
-            if db_instance.multi_az:
-                report.status = "PASS"
-                report.status_extended = (
-                    f"RDS Instance {db_instance.id} has multi-AZ enabled."
-                )
+            # Check if is member of a cluster
+            if db_instance.cluster_id:
+                for db_cluster in rds_client.db_clusters:
+                    if db_cluster.id == db_instance.cluster_id:
```

Comment:
Do you need this? You can access directly the cluster using the `cluster_id` right?

Reply:
No, because rds_client.db_clusters is a list

Reply:
could you make it as a map? it is too much to iterate over that list for each instance

Reply:
Done